monitoring/PROMETHEUS.md

```diff
@@ -0,0 +1,143 @@
+# Prometheus services Deploy - Docker compose
+
+## About
+
+This readme contains the steps for deploying ``Prometheus`` monitoring application and ``Grafana`` which is used for visualize and analyze metrics generated by Prometheus.
+
+## Disclaimer
+
+As the inclusion of services (Prometheus and Grafana) is under development, some services are currently not being monitored. The following describes the services already implemented.
+
+## Node-Exporter
+
+Node-Exporter is used for get host metrics from the host itself. In other words, the metrics obtained by Prometheus are directly related to hardware and operating system.
+
+We are currently extracting the following metrics with Node-Exporter:
+
+* Service uptime;
+* Number of CPU cores;
+* Total amount of RAM;
+* Total amount of containers;
+* Disk usage percentage;
+* RAM usage percentage;
+* Total amount of SWAP;
+* Network traffic (Sent and Received);
+* CPU usage percentage;
+* Total, free and avaliable RAM;
+* Amount of RAM for I/O processes;
+* Used space of available disks.
+
+
+## CAdvisor
+
+CAdvisor is an exporter developed by Google that provides metrics regarding resource usage and performance characteristics of running containers.
+
+We are currently extracting the following metrics with CAdvisor:
+
+
+* Current status of the container: ``Running``, ``Stopped`` and ``Paused``;
+* Network traffic received by container;
+* Network traffic sent by container;
+* Percentage of RAM usage per container;
+* Percentage of CPU usage per container;
+* RSS Memory Usage per Container
+* RAM consumption table per container;
+* Available RAM for each conatainer;
+*  RAM limit established for each container
+
+## Using the service
+
+Since we are using Docker, it is natural that we will use its metrics. To set the Docker daemon as a Prometheus target, you need to specify the metrics-address in ``/etc/docker/daemon.json``.
+
+The first step is editing the file ``/etc/docker/daemon.json``.
+
+If the file does not exists, you need to create it.
+
+If the file is empty, paste the following:
+
+``
+{
+  "metrics-addr" : "0.0.0.0:9323",
+  "experimental" : true
+}
+``
+
+If the file is not empty, add these two keys, making sure the resulting file is a valid JSON. Be careful that all lines end with a comma  "," except the last line.
```

Comment:
There is two spaces between `comma` and `","`. You should remove one.
i think you should add a `must` between `lines` and `end` like this:
```
Be careful that all lines must end...
```